test/dataframe.jl

```diff
@@ -1,31 +1,52 @@
-using DataArrays,DataFrames
+using NullableArrays,CategoricalArrays,DataFrames
 
-v110 = rcopy(DataArray,reval("x <- 1:10"))
-@test isa(v110,DataVector)
-@test eltype(v110) == Cint
+@test rcopy(Nullable, RObject(1)).value == 1
+@test rcopy(Nullable, RObject("abc")).value == "abc"
+@test rcopy(RObject(Nullable(1))) == 1
+@test rcopy(Nullable, RObject(Nullable(1, true))).isnull
```

Comment:
You should really avoid accessing the `isnull` field directly, as it's likely to go away soon (https://github.com/JuliaLang/julia/pull/18510). Use the accessor instead. (This also applies in other places.)